x-pack/plugins/security_solution/public/management/common/routing.ts

```diff
@@ -178,3 +179,15 @@ export const getTrustedAppsListPath = (location?: Partial<TrustedAppsListPageLoc
     querystring.stringify(normalizeTrustedAppsPageLocation(location))
   )}`;
 };
+
+export const getEventFiltersListPath = (
+  location?: Partial<TrustedAppsListPageLocation>
```

Comment:
Should be this type different from trusted apps? Or if we have to reuse this type maybe we should change the type name

Reply:
No, this should be different. Its my mistake (copy and paste). I'll correct it